.github/workflows/publish.yml

```diff
@@ -118,12 +118,13 @@ jobs:
       - name: Build
         run: npm run build
       - name: Publish
+        env:
+          NPM_CONFIG_USERCONFIG: ~/.npmrc
         run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN  }}" > ~/.npmrc
           npm publish --tag ${{ needs.determine-version.outputs.tag }}
           cd packages/angular/dist/swissgeol-core-angular && npm publish --tag ${{ needs.determine-version.outputs.tag }} && cd ../../../..
           cd packages/react   && npm publish --tag ${{ needs.determine-version.outputs.tag }} && cd ../..
```

Comment:
PP: Command besser strukturieren, damit die `publish` commands nicht gesucht werden müssen.

Q: Braucht es beim Publish nicht noch `--access=public.` Gemäss Doku [hier](https://docs.npmjs.com/cli/v8/commands/npm-publish#access), sind alle Scoped packages default _restricted_.

> When publishing scoped packages, the access level defaults to restricted.

PP: Provenance statements [erstelllen](https://docs.github.com/en/actions/how-tos/use-cases-and-examples/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry) `--provenance`